.github/workflows/test.yaml

```
@@ -31,4 +35,13 @@ jobs:
         if: always()
         with:
           name: Coverage results
-          path: ${{ steps.unity-test-step.outputs.coveragePath }}
\ No newline at end of file
+          path: ${{ steps.unity-test-step.outputs.coveragePath }}
+      - name: SonarCloud Scan
```

Perhaps we want to add a comment which explains this is not the optimal way to scan.
```suggestion
      # Unity does not support MSBuild, so this is the most optimal way to collect code coverage. 
      - name: SonarCloud Scan
```

---

Is it not though? To me it seems like using the Unity Test Runner and its own code coverage tool _is_ the optimal way to collect code coverage.

---

Nevermind lol, I misread

---

I mean, it wouldn't hurt, and does not impact performance. Why would we not explain our choices in the file?

---

It is the norm to not have comments in yaml pipeline files, and I would also not like to have them, names should be enough, when looking at the logs you are also not looking at the yaml files, you need to know what's happening by seeing the names.